docs/develop/go/test-suites.mdx

```diff
@@ -160,6 +161,103 @@ When an Activity sends a Heartbeat, be sure that you can see the Heartbeats in y
 
 If an Activity is supposed to react to a Cancellation, you can test whether it reacts correctly by canceling it.
 
+## Mock and override Nexus operations
+
+A workflow that executes a Nexus operation can be tested by providing a mock of the Nexus operation
+or overriding its implementation with the Nexus mock environment. This allows you to test the
+workflow without needing to create a Nexus handler service to run the Nexus operation.
+
+Let's take a look at a test that simulates a Nexus operation call. Assume the Nexus operation is
```

Comment:
Style guide suggests avoiding "we" voice (Let's). You can reword with "Consider a test that simulates..." to avoid.
Also consider s/Assume/In this example/ because that avoids directly telling the reader what to do, and minimizes the reader's need or desire to remember these roles.

I really like that you explain that it simulates a Nexus operation call _first_.